README.md

```diff
@@ -1,3 +1,25 @@
-# harness-operator
+# infra-operator
 
-th2 cluster operator
\ No newline at end of file
+Infra operator is java implementation of Kubernetes custom resource controller. It is part of _th2 infrastructure_. 
```

Comment:
Please add a little description for `custom resource` and link to full documentation